start.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+nitriding \
+	-acme \
+	-fqdn "star-randsrv.bsg.brave.software" \
```

Comment:
FYI: This is the domain for the dev environment. Production is probably going to be star-randsrv.bsg.brave.com. No need to change this; just letting you know that we're going to have to change this some time soon.